code/game/machinery/recycle_vendor.dm

```diff
@@ -636,6 +688,19 @@
 		if(materials_stored[matname] > sellthreshold)
 			sellonething(matname, materials_stored[matname] - sellthreshold) // sell excess
 		
+/obj/machinery/amesilo/proc/updatesubsidy()
+	if(my_account.money <= 400)
```

Comment:
the threshold for actually triggering the subsidy is 400c, which is more than three sheets of any currently existing materials.